compiler-rt/test/lit.common.configured.in

```diff
@@ -25,6 +25,7 @@ set_default("gold_executable", "@GOLD_EXECUTABLE@")
 set_default("clang", "@COMPILER_RT_RESOLVED_TEST_COMPILER@")
 set_default("compiler_id", "@COMPILER_RT_TEST_COMPILER_ID@")
 set_default("python_executable", "@Python3_EXECUTABLE@")
+set_default("python_root_dir", "@Python3_ROOT_DIR@")
```

Comment:
Why is this change in compiler-rt necessary?

Reply:
PR opened: https://github.com/llvm/llvm-project/pull/143738

Reply:
Not if they are cherry-picks.